lib/src/util.dart

```diff
@@ -338,3 +340,45 @@ String stripPrivateGeneratedPrefix(String value) {
       ? value.substring(privateGeneratedPrefix.length)
       : value;
 }
+
+/// Returns whether or not the source file contains a specific comment.
+///
+/// Can be used to determine whether or not the file has been modified
+/// already. This is useful (in combination with [allComments] because the
+/// visitor used for visiting comments does not work as expected, making it
+/// difficult to iterate over comments.
+bool hasComment(AstNode node, SourceFile sourceFile, String comment) {
+  final line = sourceFile.getLine(node.offset);
```

Comment:
#supernit This confused me at first glance.  Based on the variable name, I assumed this would be the source text found on that line.  That could just be me - but I think it would be more clear if this was named `lineNumber`.

Reply:
There's no harm in making it more clear, so that sounds good!